src/ui/public/index_patterns/_calculate_indices.js

```diff
@@ -7,6 +7,23 @@ define(function (require) {
     return moment.isMoment(val) ? val.valueOf() : val;
   }
 
+  // returns a properly formatted millisecond timestamp index constraint
+  function msConstraint(comparison, value) {
+    return {
+      [comparison]: timeValue(value),
```

Comment:
I should have noticed there is a fair amount of pre-existing ES6 in this file :)